manage_breast_screening/mammograms/tests/test_presenters.py

```diff
@@ -84,6 +82,17 @@ def test_last_known_screening(self, mock_appointment):
             "type": None,
         }
 
+    def test_participant_url(self, mock_appointment):
+        mock_appointment.pk = UUID("8e25aa42-9bf2-4d5d-953c-7487579d4a3b")
+        mock_appointment.screening_episode.participant.pk = UUID(
+            "ac1b68ec-06a4-40a0-a016-7108dffe4397"
+        )
+        result = AppointmentPresenter(mock_appointment)
+        assert (
+            result.participant_url
+            == "/participants/ac1b68ec-06a4-40a0-a016-7108dffe4397/?appointment_id=8e25aa42-9bf2-4d5d-953c-7487579d4a3b"
```

Comment:
Can we remove the trailing slash in the manage_breast_screening/participants/urls.py so this (`/?`) doesn't happen?

Reply:
not an issue now that we've removed the appointment_id param.